doc/how_to/custom_components/esm/panes.md

```diff
@@ -0,0 +1,178 @@
+# Create Panes with ReactiveHTML
+
+In this guide we will show you how to efficiently implement custom panes using `JSComponent`, `ReactComponent` and `AnyWidgetComponent` to get input from the user.
+
+## Creating a ChartJS Pane
+
+This example will show you the basics of creating a [ChartJS](https://www.chartjs.org/docs/latest/) pane.
+
+```{pyodide}
+import panel as pn
+import param
+from panel.custom import JSComponent
+
+
+class ChartJSComponent(JSComponent):
+    object = param.Dict()
+
+    _esm = """
+import { Chart } from "https://esm.sh/chart.js/auto"
+
+let chart = null;
+
+function createChart(canvasEl, model) {
+    if (chart) {
+        chart.destroy();
+    }
+    chart = new Chart(canvasEl.getContext('2d'), model.object);
+}
+
+export function render({ model }) {
+    const canvasEl = document.createElement('canvas');
+    // The chart will not render before after the layout has been created
+    requestAnimationFrame(() => createChart(canvasEl, model));
+
+    const updateChart = () => createChart(canvasEl, model);
+    model.on('object', updateChart);
+
+    return canvasEl;
+}
+"""
+
+
+def data(chart_type="line"):
+    return {
+        "type": chart_type,
+        "data": {
+            "labels": ["January", "February", "March", "April", "May", "June", "July"],
+            "datasets": [
+                {
+                    "label": "Data",
+                    "backgroundColor": "rgb(255, 99, 132)",
+                    "borderColor": "rgb(255, 99, 132)",
+                    "data": [0, 10, 5, 2, 20, 30, 45],
+                }
+            ],
+        },
+        "options": {
+            "responsive": True,
+            "maintainAspectRatio": False,
+        },
+    }
+
+
+chart_type = pn.widgets.RadioBoxGroup(
```

Comment:
When I click the radiobutton an exception is logged in the browser console. I can't see its related to my code.

Is it a Panel esm bug @philippjfr ?

![image](https://github.com/user-attachments/assets/a67091b2-c18e-4693-ae81-9cf1b24ca19b)

Reply:
I see the same exception with the `ReactComponent` example.

Reply:
I see the same exception with the `AnyWidgetComponent` example.

Reply:
Don't see this anymore.